boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 -o operator_name -i operator_image -c saas-repository-channel [-t]" 1>&2; exit 1; }
+
+while getopts "o:h:c:" option; do
+    case "${option}" in
+        o)
+            operator_name=${OPTARG}
+            ;;
+        c)
+            operator_channel=${OPTARG}
+            ;;
+        h)
```

Comment:
As this is already evaluated within the make file (for opm-build-push), I believe it is better to use everywhere that single value rather than having it evaluated (potentially to different values) within multiple scripts (may it be from a common lib or not).